### PR TITLE
Add more debug info for event timeline items

### PR DESCRIPTION
**Pull request by jplatte**
Yesterday, I was debugging reply edits with Stefan, this would have been very useful.

### CODE REVIEW

**Review by stefanceriu — COMMENTED** [review]
Veery nice, thanks!

File: bindings/matrix-sdk-ffi/src/timeline.rs [hunk 1]
```diff
@@ -305,7 +305,7 @@ impl EventTimelineItem {
     }
 
     pub fn raw(&self) -> Option<String> {
```
Comment (poljar) [inline bindings/matrix-sdk-ffi/src/timeline.rs]: Shouldn't we rename this one as well?

**Review by poljar — APPROVED** [review]
Looks good, left a small comment.

File: bindings/matrix-sdk-ffi/src/timeline.rs (hunk 1 above)
Comment (jplatte) [reply bindings/matrix-sdk-ffi/src/timeline.rs]: It is removed in the second commit, so I didn't bother renaming it first.

**Pull request merged**